src/gui/CMakeLists.txt

```diff
@@ -223,13 +197,41 @@ target_link_libraries(qbt_gui
         Qt5::Gui Qt5::Widgets
 )
 
-if (APPLE)
-    target_sources(qbt_gui PRIVATE macutilities.h macutilities.mm)
-    find_package(Qt5 ${requiredQtVersion} REQUIRED COMPONENTS MacExtras)
-    target_link_libraries(qbt_gui PRIVATE Qt5::MacExtras objc)
+if ((CMAKE_SYSTEM_NAME STREQUAL "Windows") OR (CMAKE_SYSTEM_NAME STREQUAL "Darwin") )
+    target_sources(qbt_gui
+        PRIVATE
+            programupdater.h
+            programupdater.cpp
+    )
+
+    if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
I would move this and the next `if`s outside of outer one.

Reply:
@glassez fixed.